index.html

```diff
@@ -1879,6 +1879,88 @@ <h3>Versioning Cryptography Suites</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Layer Cryptographic Options Appropriately</h3>
```

Comment:
This seems to be more about:

```suggestion
        <h3>Protecting and Prioritizing Application Developers</h3>
```

This section seems to weave both of these ideas together: Cryptography is about protecting data, but good cryptographic primitives and implementations also protect application developers from their (often intentional) lack of knowledge about how the cryptography works. Application developers have other things they need to focus on. Additionally, good crypto tools and application of primitives must prioritize application developer use cases -- or else the tools will ultimately not be used. This may mean that where two goals are competing with one another: crypto implementation difficulty vs. crypto tool usefulness to application developers... that application developers should be prioritized -- but not so much so that security becomes infeasible. We may want to mention that test suites for the implementation of the crypto tools (like data integrity suites) can help in these situations?

Note: Also, it's better to avoid the word "appropriately" because it can be interpreted very subjectively -- "fit for purpose" with a discussion of the intended purpose(s) might avoid this problem.